lightning/src/chain/keysinterface.rs

```diff
@@ -319,13 +319,13 @@ pub trait ChannelKeys : Send+Clone {
 	/// protocol.
 	fn sign_channel_announcement<T: secp256k1::Signing>(&self, msg: &UnsignedChannelAnnouncement, secp_ctx: &Secp256k1<T>) -> Result<Signature, ()>;
 
-	/// Set the remote channel basepoints and remote/local to_self_delay.
+	/// Set the remote channel basepoints and counterparty/local_to_self_delay.
```

Comment:
What do we think about `counterparty_set_to_local_delay` and `local_set_to_counterparty_delay`? I still have a hard time thinking that this is that much clearer.